components/message/typings.ts

```diff
@@ -16,7 +20,7 @@ export interface NzMessageDataOptions {
 
 export interface NzMessageData {
   type?: NzMessageType | string;
-  content?: string | TemplateRef<void>;
+  content?: string | TemplateRef<void | { $implicit: NzMNComponent; data: NzSafeAny }>;
```

Comment:
I think NotificationData has nothing to do with MessageData, so the change here is unnecessary. Or we can also add nzData support to message like notification. WDYT?

Reply:
NzNotificationComponent extends MessageComponent and it’s typings as well. But it’s okey to implement nzData in message component too ! Will make it in this same pr